tmva/sofie/inc/TMVA/ROperator_Reduce.hxx

```diff
@@ -0,0 +1,301 @@
+// #ifndef TMVA_SOFIE_ROPERATOR_Reduce
+// #define TMVA_SOFIE_ROPERATOR_Reduce
+
+// #include "TMVA/SOFIE_common.hxx"
+// #include "TMVA/ROperator.hxx"
+// #include "TMVA/RModel.hxx"
+
+// #include <memory>
+// #include <sstream>
+// #include <algorithm>
+// #include <stdexcept>
+// #include <vector>
+// #include <cassert>
+
+// namespace TMVA{
+// namespace Experimental{
+// namespace SOFIE{
+
+// enum ReduceOpMode { ReduceMean, ReduceSumsquare, ReduceProd };
```

Comment:
This should not me commented , you need these definitions later

Reply:
and it should be : `enum EReduceOpMode`  with an `E` in front so from the coding convention we know is an enumeration